Product-list/product-list.css

```diff
@@ -0,0 +1,50 @@
+.main-products-section {
+  display: flex;
+  margin-top: 3rem;
+}
+
+.filter-section {
+  flex: 0 0 17.5rem;
+  padding: 0px 10px 0px 0px;
```

Comment:
if possible use directional padding if padding is there only in one direction for better readability

Reply:
Right Srikar, it makes sense, thanks for suggesting it. I have updated it now. :)